fix: Removed kwargs argument as we only need to support 'qualified' v…

abhilash-kumar-nair opened the pull request:
Code snippet to try. @efredriksson-modelon  -->

```
from modelon.impact.client import Client

client = Client(url="http://localhost:8080")
workspace = client.create_workspace("Hello1")

# Choose analysis type
dynamic = workspace.get_custom_function('dynamic')

# Compile model
model = workspace.get_model("Modelica.Blocks.Examples.PID_Controller")
fmu = model.compile(compiler_options=dynamic.get_compiler_options()).wait()
# print(fmu.id)

# # Execute experiment
experiment_definition = fmu.new_experiment_definition(dynamic).with_modifiers(
    driveAngle=40
)
exp = workspace.execute(experiment_definition).wait()

```

efredriksson-modelon reviewed (changes requested):
This will be a breaking change so unless there is a mayor gain for this change, I don't think we should do this.

If there is a usability problem, we should update the documentation. If we really don't like this form of giving modifiers, we could remove all traces of it from the documentation but keep the implementation. A middle way could be to ensure the examples all use the preferred way, so the reader only need to take in one way of specifying modifiers.

abhilash-kumar-nair commented:
> This will be a breaking change so unless there is a mayor gain for this change, I don't think we should do this.
> 
> If there is a usability problem, we should update the documentation. If we really don't like this form of giving modifiers, we could remove all traces of it from the documentation but keep the implementation. A middle way could be to ensure the examples all use the preferred way, so the reader only need to take in one way of specifying modifiers.

@efredriksson-modelon, I have removed **kwargs** from the docs and also added a deprecation warning. However I have still kept the usage in the test,  we have for the 'with_modifier' function. I feel we should have them tested until we finally deprecate them in a future release. What do you think?

efredriksson-modelon commented on modelon/impact/client/experiment_definition.py:
```diff
@@ -1,8 +1,13 @@
 from modelon.impact.client import exceptions
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```
remove

efredriksson-modelon commented on modelon/impact/client/experiment_definition.py:
```diff
@@ -240,6 +239,12 @@ def with_modifiers(self, modifiers=None, **modifiers_kwargs):
                 'inertia2.J': Range(0.1, 0.5, 3)})
         """
+        if modifiers_kwargs:
+            logger.info(
```
info -> warning

efredriksson-modelon commented on modelon/impact/client/experiment_definition.py:
```diff
@@ -468,6 +467,12 @@ def with_modifiers(self, modifiers=None, **modifiers_kwargs):
 
         """
+        if modifiers_kwargs:
+            logger.info(
```
info -> warning

efredriksson-modelon reviewed (changes requested):
We also need to update the sphinx documentation.

efredriksson-modelon reviewed (approved):
👍
